src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -239,7 +240,7 @@ public GetSlaveDigest(FilePath rootPath) {
         public String call() {
             StringBuilder result = new StringBuilder();
             final File rootPath = new File(this.rootPathName);
-            for (File file : rootPath.listFiles()) {
+            for (File file : Preconditions.checkNotNull(rootPath.listFiles())) {
```

Comment:
Throwing an `IllegalArgumentException` here is not really an improvement over a `NullPointerException` I think.